### ISSUE TITLE
DNS has only partial support

### ISSUE THREAD

**Issue by jprestwo** [Samsung/libtuv#25]
DNS is missing resolve functionality. There is no implementation for:

**Comment by seanshpark** [comment]
Yes, including this function, there may be lots of missing compared to libuv.
